nilearn/plotting/img_plotting.py

```diff
@@ -1379,6 +1379,11 @@ def plot_carpet(img, mask_img=None, mask_labels=None,
         If ``mask_img`` corresponds to an atlas, then this dictionary maps
         values from the ``mask_img`` to labels. Dictionary keys are labels
         and values are values within the atlas.
+    %(t_r)s
+
+        .. note::
+            If ``t_r`` is not provided, it will be inferred from ``img``'s
+            header.
```

Comment:
I'm not sure if this addendum merits an admonition, but I took my cue from the `cmap` documentation below.